packages/design-system/src/themes/default/colors.css

```diff
@@ -97,5 +97,11 @@
   --ads-old-color-royal-blue: #457AE6;
   --ads-old-color-silver: #C4C4C4;
   --ads-old-color-dove-gray-3: #6F6F6F;
+
+  --ads-old-color-black-750-opacity-50: #4b484880;
```

Comment:
That is not a common approach to do here... and I don think we can do string interpolation in css

Reply:
We can't do it in css, but we can do it in styled, and we use styled everywhere for css, so that's what I was thinking.
I think the common approach is to use rgba and an opacity variable as described here: https://stackoverflow.com/questions/40010597/how-do-i-apply-opacity-to-a-css-color-variable. Of course, not valid for this PR at this point but perhaps add a TODO that we should refactor this in phase 2?